web/topicRegistry/data-and-apis.json

```diff
@@ -21,7 +21,7 @@
           "owner": "bcgov",
           "title": "BC Government API Registry",
           "description": "The Province of British Columbia API Registry",
-          "image": "https://developer.gov.bc.ca/images/gov-logo-static.png"
+          "image": "https://classic.developer.gov.bc.ca/images/gov-logo-static.png"
```

Comment:
ohhh, it was the image location changed, good catch!

Reply:
Thanks! Took some digging around :)